timestamp-impl/src/main/java/com/palantir/timestamp/DebugLogger.java

```diff
@@ -37,22 +40,23 @@ private DebugLogger() {
 
     public static void handedOutTimestamps(TimestampRange range) {
         long count = range.getUpperBound() - range.getLowerBound() + 1L;
-        logger.trace("Handing out {} timestamps, taking us to {}.", count, range.getUpperBound());
+        logger.trace("Handing out {} timestamps, taking us to {}.", SafeArg.of("count", count),
+                SafeArg.of("rangeUpperBound", range.getUpperBound()));
     }
 
     public static void createdPersistentTimestamp() {
         logger.info("Creating PersistentTimestamp object on thread {}."
                         + " If you are running embedded AtlasDB, this should only happen once."
                         + " If you are using Timelock, this should happen once per client per leadership election",
-                Thread.currentThread().getName());
+                UnsafeArg.of("threadName", Thread.currentThread().getName()));
```

Comment:
Right because this thread was created by the application consuming atlasdb, I couldn't guarantee that it was always safe so it must be unsafe